Website/src/rendering/src/components/SpeakersGrid.tsx

```diff
@@ -1,74 +1,44 @@
-import speaker7 from '../../data/media/img/speakers/kate-green.jpeg';
-import speaker8 from '../../data/media/img/speakers/jalen-williams.jpeg';
+type Speaker = {
+  fields: {
+    Name: Field<string>;
+    Role: Field<string>;
+    Image: ImageField;
+  };
+};
 
-const SpeakersGrid = (): JSX.Element => (
-  <div className="section__speakers__grid">
-    <Link href="/speakers/mary-asada">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker1} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Mary Asada</p>
-        <p className="speaker__role">Athlete</p>
-      </a>
-    </Link>
-    <Link href="/speakers/martinmoore">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker2} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Martin Moore</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/jacobgonzalez">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker3} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Jacob Gonzalez</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/edjones">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker4} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Ed Jones</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/sophiataylor">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker5} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Sophia Taylor</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/lixiuying">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker6} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Li Xiu Ying</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/kategreen">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker7} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Kate Green</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-    <Link href="/speakers/jalenwilliams">
-      <a className="section__speakers__grid__speaker">
-        <Image src={speaker8} alt="Speaker" width={260} height={260} />
-        <p className="speaker__name">Jalen Williams</p>
-        <p className="speaker__role">Speaker</p>
-      </a>
-    </Link>
-  </div>
-);
+type SpeakersGridProps = ComponentProps & {
+  fields: {
+    items: Speaker[];
+  };
+};
+
+const SpeakersGrid = (props: SpeakersGridProps): JSX.Element => {
+  console.log(props.fields);
```

Comment:
Please pay attention not to commit debugging code by reviewing changed files line by line before adding them to the files to commit.